smartling-job-batches-api/src/main/java/com/smartling/api/jobbatches/v1/pto/BatchItemPTO.java

```diff
@@ -0,0 +1,21 @@
+package com.smartling.api.jobbatches.v1.pto;
+
+import com.smartling.api.v2.response.ResponseData;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.Calendar;
+import java.util.List;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+public class BatchItemPTO implements ResponseData
```

Comment:
The most of PTOs are duplicated between v1 and v2, and we are not going to reuse common classes
These PTOs might become incompatible in the future

Reply:
👍